contrib/jinzhu/gorm/gorm.go

```diff
@@ -0,0 +1,18 @@
+// Package gorm provides helper functions for tracing the jinzhu/gorm package (https://github.com/jinzhu/gorm).
+package gorm
+
+import (
+	sqltraced "gopkg.in/DataDog/dd-trace-go.v1/contrib/database/sql"
+
+	"github.com/jinzhu/gorm"
+)
+
+// Open opens a new (traced) database connection. The used dialect must be formerly registered
+// using (gopkg.in/DataDog/dd-trace-go.v1/contrib/database/sql).Register.
+func Open(dialect, source string) (*gorm.DB, error) {
```

Comment:
`gorm.Open` supports both a database string or passing the actual database through... I was wondering if we should try to do the same, but I guess there's really no value in supporting both since there's no way to take an existing db and add tracing too it?

Reply:
Good point. I will try and investigate it.